### PR TITLE
Enable warnings in local development mode

### PR DESCRIPTION
**Pull request by jamescrowley**
This PR is enabling PHP warnings for local development, so we can slowly get the code base to a point there are no warnings. Having the warnings enabled should save us some occasional 'gotchas' with variable scopes and declarations - already found one bug whilst creating this PR having turned on the warnings

### CODE REVIEW

**Comment by HaGuesto** [comment]
@jamescrowley What is this PR for?

**Comment by jamescrowley** [comment]
Conflicts resolved :)

File: templates/start-market.tpl [hunk 1]
```diff
@@ -150,7 +150,7 @@ var chart = AmCharts.makeChart( "chartdiv2", {
     <hr />
       {$data['weeklabel']} week, {if $data['newpeople']}<span class="number">{$data['newpeople']}</span>{else}no{/if} new {if $data['newpeople']==1} beneficiary was {else} beneficiaries were {/if} registered in {$currentcamp['name']}.
  			{if isset($data['notregistered'])}
-  				Additionally there {if $data['nonregistered']==1}is {else} are {/if} <span class="number">{$data['notregistered']}</span> unregistered {if $data['notregistered']==1} person{else} people{/if}.
+  				Additionally there {if isset($data['nonregistered'])}is {else} are {/if} <span class="number">{$data['notregistered']}</span> unregistered {if $data['notregistered']==1} person{else} people{/if}.
```
Comment (HaGuesto) [inline templates/start-market.tpl]: This one is weird. Two lines above the index is called 'notregistered' and here it is 'nonregistered'.

Comment (jamescrowley) [reply templates/start-market.tpl]: Fixed

**Pull request merged**